modules/api/pkg/handler/v2/cluster_default/cluster_default.go

```diff
@@ -254,17 +254,33 @@ func initializeCloudProviderSpec(dc string, provider kubermaticv1.ProviderType)
 	// We intentionally keep the cloud spec empty in the default cluster spec since it mostly depends on the cloud credentials.
 	switch provider {
 	case kubermaticv1.AWSCloudProvider:
-		cloudSpec.AWS = &kubermaticv1.AWSCloudSpec{}
+		cloudSpec.AWS = &kubermaticv1.AWSCloudSpec{
+			NodePortsAllowedIPRanges: &kubermaticv1.NetworkRanges{
+				CIDRBlocks: []string{networkDefaults.IPv4.NodePortsAllowedIPRange, networkDefaults.IPv6.NodePortsAllowedIPRange},
```

Comment:
This looks like a hack to me. Especially if the comment above mentions `we intentionally keep the cloud spec empty`.
I think this should be defaulted properly in `defaulting.DefaultClusterSpec` instead.